static/sass/_pattern_buttons.scss

```diff
@@ -1,4 +1,33 @@
 @mixin canonical-p-buttons {
+  .p-button--accordion {
+    background-color: inherit;
+    border: none;
+    background: {
+      position: top 50% left $sph-inner;
+      repeat: no-repeat;
+    }
+    background-color: inherit;
+    border: 0;
```

Comment:
the `border` and `background-color` rules are repeated here, and I think the `background-color` rule could be merged into the `background` object on line 5.